### DISCUSSION TITLE
Issue with the `BindTapGesture()` method in v4.2.0 on .NET 9 RC2, whereas v4.1.0 works fine

### DISCUSSION THREAD

**Discussion by egvijayanand** [CommunityToolkit/Maui.Markup#329] [General]
There is an issue with the `BindTapGesture()` method in v4.2.0 on .NET 9 RC2, whereas v4.1.0 works fine.

Since it's a preview version, I am not logging an issue but rather making you aware of this @brminnick 

Error message: 

```
Method 'ApplyToResolvedSource' in type 'CommunityToolkit.Maui.Markup.TypedBinding`2' from assembly 'CommunityToolkit.Maui.Markup, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
```

**Comment by TheCodeTraveler** [comment]
Got it. Yup - bindings are totally different under the hood in .NET 9

**Reply by egvijayanand** [reply]
Most of the other things work fine, but only this method is cribbing.

**Comment by egvijayanand** [comment]
Adding context to the issue, am using TypedBinding.

**Comment by egvijayanand** [comment]
@brminnick Since classic Gesture methods have been removed in v5.0,

What is the support plan for RelativeBindingSource?

I have templates defined in multiple places that rely solely on RelativeBindingSource for gestures.

I tried upgrading to Markup Toolkit v5.0 and immediately encountered this issue.

Do you have any samples that handle it differently?

**Reply by TheCodeTraveler** [reply]
Ha- I've already re-added them! https://github.com/CommunityToolkit/Maui.Markup/commit/205b55fc1ec5ee7ec7d99818a1d2e3189c7c452f

I shouldn't have removed them. They'll be back in the next release. 

@jfversluis Are you able to publish `CommunityToolkit.Maui.Markup v5.1.0` this week?

**Reply by jfversluis** [reply]
Building now! If you could please add some release notes: https://github.com/CommunityToolkit/Maui.Markup/releases/tag/5.1.0

**Reply by TheCodeTraveler** [reply]
Thanks Gerald!! I'm on it 🙌

**Reply by egvijayanand** [reply]
Thanks for the quick turnaround.

Since the type name is now updated to `StringGesturesExtensions` instead of `GesturesExtensions`, a reference error may occur. Please mention it in the [changelog](https://github.com/CommunityToolkit/Maui.Markup/releases/tag/5.1.0).

**Reply by egvijayanand** [reply]
@brminnick Thanks for updating the changelog.

But needs a quick correction: the sample code mentioned needs an update, as it is missing parentheses and has a parameter name mismatch.

Here is the modified version:

```cs
StringGesturesExtensions.BindTapGesture(myLabel, nameof(TrendsViewModel.TappedCommand), commandSource: new RelativeBindingSource(RelativeBindingSourceMode.FindAncestorBindingContext, typeof(TrendsViewModel)));
```

**Reply by TheCodeTraveler** [reply]
Thanks again Vijay! Done ✅
